### PR TITLE
New convert project to grant email

### LINKED ISSUE #7671: As a hackathon project buidler, I want to know that I can convert my project to grant
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As a hackathon project buidler, I want to know that I can convert my project to grant

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: We're going to have the convert to grant feature soon, and we would like to let hackathon project buidlers know they can convert their projects to grant through a one-time announcement email.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Current Behavior
[comment]: # (Describe what actually happened.)
No email

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
Email to hackathon project builders
convert to grant email

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)
- [ ] Email template built

### Data Requirements
[comment]: # (How will we measure the success of this feature? What kind of tracking is needed for this feature (clicks, impressions, flag)?)

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)

### PR DESCRIPTION
**Pull request by sebastiantf**
##### Description
This PR builds a new email template for the email to let hackathon project buidlers know they can convert their projects to grant through a one-time announcement email.

##### Refers/Fixes
Fixes #7671 

- [x] Email template built

##### Screenshots
**Gmail Web:**

convert_project_grant_email_gmail_web

**Gmail Mobile:**

convert_project_grant_email_gmail_mobile

**Web Preview:**

convert_project_grant_email_web

### CODE REVIEW

File: app/retail/templates/emails/tribe_hackathon_prizes.html [hunk 1]
```diff
@@ -31,21 +31,9 @@
     margin-top: 30px;
     background-color: #000000;
   }
-  .left {
-    text-align: left;
-  }
```
Comment (thelostone-mc) [inline app/retail/templates/emails/tribe_hackathon_prizes.html:36]: ^ I have a hunch there are other emails which also use this prop
Could we remove those as well ?

Comment (sebastiantf) [reply app/retail/templates/emails/tribe_hackathon_prizes.html:36]: searching the entire project for `.left` only returns `template.html`

Screenshot 2020-10-20 at 12 20 18 PM

File: app/retail/templates/emails/convert_project_grant.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Get funding for your project by converting to Grant!
+
+    Do you know that a lot of important companies in the Web3 space currently started out as hackathon projects?
+
+    At Gitcoin, we believe every hackathon project is the start of something big. To get you started on your journey, now you're able to convert your project to grant!
+
+    Simply go to your project, click the Convert to Grant button, and you'll be able to get funding from contributors and be eligible for CLR matching on Gitcoin Grants Rounds!
+
+  View My Projects
```
Comment (danlipert) [inline app/retail/templates/emails/convert_project_grant.txt:9]: Whats up with this line?

**Review by danlipert — CHANGES_REQUESTED** [review]

File: app/retail/templates/emails/convert_project_grant.txt (hunk 2 above)
Comment (sebastiantf) [reply app/retail/templates/emails/convert_project_grant.txt:9]: It is only the .txt for the template .HTML with just the text

**Review by willsputra — APPROVED** [review]

**Comment by willsputra** [comment]
design looks good to me!

**Review by PixelantDesign — APPROVED** [review]

**Comment by sebastiantf** [comment]
^^ bump

**Comment by PixelantDesign** [comment]
@sebastiantf could we make the button View Project instead of View Projects?

**Comment by sebastiantf** [comment]
> @sebastiantf could we make the button View Project instead of View Projects?

@PixelantDesign done:

**Review by danlipert — APPROVED** [review]

**Comment by danlipert** [comment]
Looks good, how does it get sent out though? How do we trigger it to send?

**Comment by sebastiantf** [comment]
Management command is not yet created as the feature is not yet implemented and only the template is specified in the issue: #7671

**Comment by danlipert** [comment]
@sebastiantf fair enough!

**Review by frankchen07 — APPROVED** [review]
lgtm

**Comment by PixelantDesign** [comment]
@octavioamu could we get this one through when Convert to Grant Is ready from @sanchaymittal

**Comment by PixelantDesign** [comment]
Hi Sebastian! Thank you for building the template. Looking great!  We do need the email to work. The email should be sent out after the Hackathon is over. Everyone that submitted a project should receive this email. One email per submission.

Sorry for the confusion. Are you able to help finish this up?

**Comment by sebastiantf** [comment]
> Hi Sebastian! Thank you for building the template. Looking great! We do need the email to work. The email should be sent out after the Hackathon is over. Everyone that submitted a project should receive this email. One email per submission.
> 
> Sorry for the confusion. Are you able to help finish this up?

Sure @PixelantDesign. I add that too

**Comment by sebastiantf** [comment]
@PixelantDesign @danlipert Mgmt command is added. Apologies for the dalay

**Comment by PixelantDesign** [comment]
Thank you @sebastiantf are you able to post a video to show how it is working?

**Comment by sebastiantf** [comment]
> Thank you @sebastiantf are you able to post a video to show how it is working?

@PixelantDesign There aren't many visible things going on with the mgmt command that need to be captured in a video.

If you issue the command `manage.py convert_project_grant_email`:

1. It looks for hackathons that ended last day
2. Looks for projects submitted for those hackathons
3. Fetches the profiles associated with those projects
4. Sends out emails to each of those profiles/emails

The cron job is set to run this command at 12:15 AM every day

**Comment by thelostone-mc** [comment]
@frankchen07 is this something we can get in / do we wanna keep this open for now ?

**Comment by frankchen07** [comment]
we can archive this for now, this is not a priority

**Comment by thelostone-mc** [comment]
Closing this out based on above comment

**Pull request closed without merging**